modules/ROOT/pages/sftp/sftp-documentation.adoc

```diff
@@ -106,189 +110,233 @@ An FileSystemProvider which provides instances of SftpFileSystem from instances
 | Configuration | String | The name of the configuration to use. | | *x*{nbsp}
-| Source Path a| String |  +++The path to the file to be copied.+++ |  | *x*{nbsp}
-| Target Path a| String |  +++The target directory where the file is going to be copied.+++ |  | *x*{nbsp}
-| Create Parent Directories a| Boolean |  +++Whether or not to attempt creating any parent directories which don't exist.+++ |  +++true+++ | {nbsp}
-| Overwrite a| Boolean |  +++Whether or not to overwrite the file if the target destination already exists.+++ |  +++false+++ | {nbsp}
-| Rename To a| String |  +++Copied file's new name. If not provided, original file name is kept.+++ |  | {nbsp}
+| Source Path a| String |  The path to the file to copy. |  | *x*{nbsp}
+| Target Path a| String |  The target directory for where to copy the file. |  | *x*{nbsp}
+| Create Parent Directories a| Boolean |  Whether or not to attempt creating any parent directories that don't exist. |  true | {nbsp}
+| Overwrite a| Boolean |  Whether or not to overwrite the file if the target destination already exists. |  false | {nbsp}
+| Rename To a| String |  Rename the copied file. If this value is not provided, the original file name will be kept. |  | {nbsp}
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  | {nbsp}
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | {nbsp}
 |===
 
 
-==== For Configurations
+==== For Configurations.
 * <<config>> {nbsp}
 
 ==== Throws
 * SFTP:RETRY_EXHAUSTED {nbsp}
 * SFTP:ILLEGAL_PATH {nbsp}
-* SFTP:CONNECTIVITY {nbsp}
 * SFTP:FILE_ALREADY_EXISTS {nbsp}
+* SFTP:CONNECTIVITY {nbsp}
 
 
 [[createDirectory]]
 === Create Directory
 `<sftp:create-directory>`
 
-+++
-Creates a new directory on directoryPath.
-+++
+
+Creates a new directory on `directoryPath`.
+
 
 ==== Parameters
 [cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | *x*{nbsp}
-| Directory Path a| String |  +++The new directory's name.+++ |  | *x*{nbsp}
+| Directory Path a| String |  The new directory's name. |  | *x*{nbsp}
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  | {nbsp}
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | {nbsp}
 |===
 
 
-==== For Configurations
+==== For Configurations.
 * <<config>> {nbsp}
 
 ==== Throws
+
 * SFTP:RETRY_EXHAUSTED {nbsp}
-* SFTP:ACCESS_DENIED {nbsp}
 * SFTP:ILLEGAL_PATH {nbsp}
-* SFTP:CONNECTIVITY {nbsp}
+* SFTP:ACCESS_DENIED {nbsp}
 * SFTP:FILE_ALREADY_EXISTS {nbsp}
+* SFTP:CONNECTIVITY {nbsp}
 
 
 [[delete]]
 === Delete
 `<sftp:delete>`
 
-+++
-Deletes the file pointed by path, provided that it's not locked
-+++
+Deletes the file that the path points to, provided that it's not locked.
```

Comment:
Should be:
Deletes the file that the **Path** field points to, provided that the file is not locked.